src/Common/src/Interop/Windows/BCrypt/Interop.BCryptImportKey.cs

```diff
@@ -0,0 +1,57 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
FYI: You seem to be missing .csproj edits that would add this file to the compilation.